guides/source/autoloading_and_reloading_constants.md

```diff
@@ -1313,3 +1313,75 @@ class C < BasicObject
   end
 end
 ```
+
+### Missing Singleton Methods
```

Comment:
The particular example given here involves a missing singleton method, but it's just a symptom of the problem. If instance methods were defined on `Blog` and it was used as a mixin, the symptom would instead be missing instance methods. If nested constants were defined in `blog.rb`, the symptom would be missing constants.

Maybe "Reusing constants as namespaces"?